### DISCUSSION TITLE
How to trigger post processing via script?

### DISCUSSION THREAD

**Discussion by reconman** [pymedusa/Medusa#9023] [Q&A]
The documentation talks about 3 different ways on how to start post processing:

* Scheduled
* Manual
* Script based

The first 2 methods are documented, but I can't find anything about the 3rd.

I would like to do what the person in #6371 asked for.

**Comment by p0psicles** [comment]
You should look into nzbToMedua. And search for the wiki page for your client

**Reply by reconman** [reply]
I looked through nzbToMedia and it's too big and convoluted, especially if you want to run it in an existing docker container without building your own image.

I think I was able to reproduce nzbToMedia's behavior with this curl call:
```
#!/bin/bash
curl -X POST -d "proc_dir=/Data/Downloads/Torrents/Completed&process_method=hardlink&delete_on=on" http://192.168.0.2:8081/home/postprocess/processEpisode
```

Medusa would offer another parameter for specifying file names in the folder, but that's something I'll look into the future.
